src/Transformer.ts

```diff
@@ -149,7 +154,13 @@ export class Transformer {
       return this.convertExportDeclaration(node);
     }
     if (ts.isModuleDeclaration(node)) {
-      return this.convertNamespaceDeclaration(node);
+      if (isNamespaceDeclaration(node)) {
+        return this.convertNamespaceDeclaration(node);
```

Comment:
we would need to at least walk all the children of module declarations, so we correctly pull in any referenced items.